src/components/pages/Project.vue

```diff
@@ -0,0 +1,147 @@
+<template>
+    <div class="gradientBackground">
+        <div class="titleHeader text-center">
+            <h1 class="display-3 hidden-sm-and-down font-weight-thin white--text ">
+                {{project.type === 'donate' ? 'Spendenseite' : 'Betriebsseite'}}
+            </h1>
+            <h1 class="display-1 hidden-md-and-up font-weight-thin white--text ">
+                {{project.type === 'donate' ? 'Spendenseite' : 'Betriebsseite'}}
+            </h1>
```

Comment:
https://vuetifyjs.com/en/customization/breakpoints/ maybe?

Reply:
REVERSE! YOU WILL FIX IT!
![REVERSE](https://a.wattpad.com/cover/201516290-352-k392254.jpg)

Reply:
ok hoffentlich zu ihrer zufriedenheit gefixt